wpimath/src/main/java/edu/wpi/first/math/geometry/Translation3d.java

```diff
@@ -306,6 +311,11 @@ public Translation3d interpolate(Translation3d endValue, double t) {
         MathUtil.interpolate(this.getZ(), endValue.getZ(), t));
   }
 
+  @Override
+  public Translation3d flip(SymmetryStrategy strategy) {
+    return new Translation3d(strategy.flipX(m_x), strategy.flipY(m_y), m_z);
+  }
```

Comment:
Putting this in the geometry class feels like a violation of separation of concerns.

Reply:
The intent was to help discoverability, having methods on the type itself is the best way to expose it to new users IMO. If removing the changes to the geometry classes is needed for this to be merged I can do that.